bin/nightly_build.py

```diff
@@ -288,16 +288,17 @@ def execute_makefile(self, makefile_path=None, target=None):
         makefile_location = self.cvw.joinpath(makefile_path)
         os.chdir(makefile_location)
 
-        output_file = self.log_dir.joinpath(f"make-{target}-output.log")
+        if target: 
+            output_file = self.log_dir.joinpath(f"make-{target}-output.log")
+        else: output_file = self.log_dir.joinpath(f"make-output.log")
 
-        command = ["make"]
+        command = ["make", "-j6"]
```

Comment:
Any reason to hardcode 6 jobs? For most processes I have been using `--jobs $(nproc --ignore 1)` to use one less thread than the system has. This is more portable to different servers with different hardware.

If we want to use fewer jobs during nightly that is fine, but even then it might make sense to do something like the number of cores divided by 2 or something.

Reply:
6 cores was chosen entirely arbitrarily. cores divided by 2 makes more sense. any ideas on how to accomplish that?

Reply:
@slmnemo Were you planning to change this too or should I merge?

Reply:
i'm making the change right now, sorry for the delay.